### ISSUE TITLE
Make the debugger used a configuration option

### ISSUE THREAD

**Issue by baronfel** [formulahendry/vscode-dotnet-test-explorer#299]
It would be nice to be able to configure the debugger that's launched [here](https://github.com/formulahendry/vscode-dotnet-test-explorer/blob/master/src/debug.ts#L36) with coreclr-debugger options.

This is relevant because currently this extension can't be used for debugging in non-proprietary distributions of VSCode-compatible services like GitPod or Theia IDE.

For example, Samsung has implemented a compatibly-licensed .net core debugger that's a drop-in replacement for `vsdbg` in their [netcoredbg](https://github.com/Samsung/netcoredbg) project. If a user has this binary available (for example by downloading it from that project's releases), then they can use debugger configuration to point VSCode/VSCodium/GitPod at that binary using some [configuration parameters](https://github.com/OmniSharp/omnisharp-vscode/wiki/Attaching-to-remote-processes#configuring-ssh-attach-with-launchjson).

So long story short, it would be wonderful to be able to provide a (sub)set of the parameters for a particular debug configuration block as configuration to this extension and have that merged in with the configuration.

Example usage:

```json
"dotnet-test-explorer.extraDebugSettings": {
	"pipeTransport": {
        "pipeProgram": "bash",
        "pipeArgs": [ "-c" ],
        "debuggerPath": "~/vsdbg/vsdbg",
        "pipeCwd": "${workspaceRoot}",
        "quoteArgs": true
    },
}
```

and this would be read and splatted together with the args provided by this extension, in such a way that args provided by this extension take precedence.

**Comment by jorgepsmatos** [comment]
Any news about this topic?
